_posts/2024-04-11-time-travel-delta-lake.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: post
+title: "Time travel in Delta Lake connector"
+author: Yuya Ebihara
+excerpt_separator: <!--more-->
+image: /assets/images/logos/trino-delta.png
+show_pagenav: true
+---
+
+Exciting news - time travel capability has finally arrived in the Delta Lake
+connector! After introducing support for time travel in the Iceberg connector
+back in 2022, we're thrilled to announce that the Delta Lake connector now joins
+the ranks as the second connector offering this feature.
+
+<!--more-->
+
+## Background and motivation
+
+Time travel as a feature has a number of practical use cases:
+
+1. **Auditing and compliance**: Time travel enables auditors and compliance
+   teams to analyze data changes over time, ensuring regulatory compliance and
+   providing transparency into data operations.
+2. **Historical analysis**: Data analysts and data scientists can perform
+   historical analysis by querying data at different points in time, uncovering
+   trends, patterns, and anomalies that may not be apparent in current data.
+3. **Data recovery and rollback**: In the event of data corruption or erroneous
```

Comment:
I thinks this should be #1... probably the most generally applicable.

Reply:
yeah .. good point